docs/man/man1/skonfig.rst

```diff
@@ -20,6 +20,7 @@ SYNOPSIS
       -V          print version
       -d          print dumped hosts, -d <host> = print dump
       -i path     initial manifest or '-' to read from stdin
+      -j jobs     maximum number of jobs (by default host CPU count)
```

Comment:
I find the `by default host CPU count` confusing and agree with @fancsali. I tend to interpret it as "uses N jobs if no -j is set", not as "-j means N"

How about:
> -j [jobs]  maximum number of jobs (defaults to 1, if jobs is omitted the host CPU count is used)

Reply:
All flags are optional. Maybe it's not too obvious... :thinking:

Reply:
I initially misunderstood skonfig's `-j` option for an equivalent to cdist's.
But since the idea is to enable parallelisation by default, the comment makes perfect sense.
Ignore my comment above.